### ISSUE TITLE
Add Deconflicted List of Performance IDs to Wiki Component Development Page

### ISSUE THREAD

**Issue by msuder** [nasa/nos3#136]
https://github.com/nasa/nos3/wiki/NOS3-Component-Development

This page provides a nice, deconflicted list of cFS message ids allocated to the various nos3 provided components.

It would be nice for it to provide the same for cFS performance ids.

**Comment by jlucas9** [comment]
Does cFS 7.0 use perf ids still? The tools are nice to use if they still exist

**Comment by kevincbruce** [comment]
I'd be happy to look into it. Could use a refresher on perf IDs. Also might change the name of that page to _'Component Directory'_ or something like that, since it is more a directory or list of components and IDs than it is a development guide (the Component Development Flow page _is_ a dev guide meanwhile)

**Comment by kevincbruce** [comment]
Okay, went through manually today. Added perf_ids to the components, and made a section for CFS apps and added Message IDs and Perf Ids there, as applicable. Let me know what you think. Also, are there Perf IDs for Cryptolib? I couldn't find them, nor the message IDs that were already referenced. I'll also check through to make sure the MIDs that were listed for components are accurate.

**Comment by kevincbruce** [comment]
Due to offset 30+x and 40+x, CF still collides, but this should be resolved for all unique, set IDs, and the wiki should be up to date to match the current reality once the branch is merged. Issue is worth reviewing with the meetings around and after the next update, when we look at updating CF.

**Comment by kevincbruce** [comment]
And confirmed that it builds and seems to work correctly with the updated list.

**Comment by jlucas9** [comment]
Merged into `dev`!
